payments-core/src/main/java/com/stripe/android/model/ElementsSession.kt

```diff
@@ -52,6 +52,9 @@ data class ElementsSession(
     val enableLinkInSpm: Boolean
         get() = flags[Flag.ELEMENTS_ENABLE_LINK_SPM] == true
 
+    val allowLinkDefaultOptIn: Boolean
+        get() = linkSettings?.linkFlags?.get("link_mobile_disable_default_opt_in") != true
```

Comment:
if the flag is not present, `allowLinkDefaultOptIn` will be true, let's make sure it gets deployed before shipping!